02 - Community Expansions/02.01 Plugins by Category/Automation plugins.md

```diff
@@ -7,34 +7,33 @@ publish: true
 ---
 
 
-# Automation
+# Automation plugins
 
-Automate create, update, delete of notes
+Automatically create, update, or delete notes.
 
 ## Plugins in this category
 
+- [[templater-obsidian|Templater]]
 - [[snippets|Snippets plugin]]
 - [[obsidian-text-expander|Text Expander]]
-- [[obsidian-paper-cut|PaperCut]]
 - [[text-snippets-obsidian|Text Snippets]]
 - [[buttons|Buttons]]
 - [[python-lab-plugin|Python lab plugin]]
 - [[obsidian-qrcode-plugin|QR Code Generator Plugin]]
 - [[oz-clear-unused-images|Clear Unused Images]]
-- [[obsidian-react-components|React Components]]
+- [[obsidian-react-components|React Components]] #category/review-classification 
```

Comment:
Was the added `#category/review-classification` a typo?

Reply:
Oh, no it wasn't. I added that tag to plugins I think are misclassified. I probably didn't get to all of them